client/snapper.cc

```diff
@@ -356,6 +356,7 @@ help_list()
 	 << _("    Options for 'list' command:") << '\n'
 	 << _("\t--type, -t <type>\t\tType of snapshots to list.") << '\n'
 	 << _("\t--disable-used-space\t\tDisable showing used space.") << '\n'
+	 << _("\t--no-qgroup-rescan\t\tDo not rescan groups.") << '\n'
```

Comment:
Please use a generic name, e.g. no-used-space-rescan. snapper is not only about btrfs and other storage systems/file systems might also need that option.

Reply:
I've renamed to option to be generic and consistent with the disable option and documented it in the referenced file.